integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -12,7 +12,7 @@ context('Create Barcode Image', () => {
     Page.verifyOnPage(ChooseBarcodeOptionPage).continueToImage()
     Page.verifyOnPage(GenerateBarcodeImagePage)
       .barcodeAddressImageExists()
-      .imageDownloadButtonExists('Gage-Hewitt-A1234BC.png')
+      .imageDownloadButtonExists(/Gage-Hewitt-A1234BC-\d{2}-\d{2}-\d{4}\.png/)
```

Comment:
From a conversation on the original testing of the barcode images I've added a date suffix to the barcode image filename. It'll keep KH happy!